xml/System.Security.Cryptography/IncrementalHash.xml

```diff
@@ -120,11 +123,21 @@
         <Parameter Name="count" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="data">To be added.</param>
-        <param name="offset">To be added.</param>
-        <param name="count">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="data">The data to process.</param>
+        <param name="offset">The offset into the byte array from which to begin using data.</param>
+        <param name="count">The number of bytes to use from <c>data</c>.</param>
+        <summary>Appends the specified number of bytes from the specified data, starting at the specified offset, to the data already processed in the hash or Hash-based Message Authentication Code (HMAC).</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="data" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ArgumentOutOfRangeException">
+          <paramref name="count" /> or <paramref name="offset" /> is negative.  
```

Comment:
According to our guidelines, you can describe the same condition in one sentence. 
The (newline) -or- (newline) would be for different triggering condition. Makes sense?